presidio-anonymizer/presidio_anonymizer/entities/anonymizer_config.py

```diff
@@ -0,0 +1,54 @@
+"""Handle the anonymizers data - anonymizer class and params."""
+import logging
+
+from presidio_anonymizer.anonymizers import Anonymizer
+from presidio_anonymizer.entities import InvalidParamException
+
+
+class AnonymizerConfig:
+    """Handle the anonymizers data - anonymizer class and params."""
+
+    @classmethod
+    def from_json(cls, params: dict):
+        """
+        Create AnonymizerConfig from json.
+
+        :param params: json with struct of {
+            "type": "mask",
+            "masking_char": "*",
+            "chars_to_mask": 4,
+            "from_end": true
+        }
+        :return: AnonymizerConfig
+        """
+        anonymizer_name = params.get("type")
+        if anonymizer_name:
+            params.pop("type")
+        return cls(anonymizer_name, params)
+
+    def __init__(self, anonymizer_name: str, params: dict = {}):
```

Comment:
Better not to instantiate params with {} but with None, and create an empty dict if None (and if required)
Default argument value shouldn't be mutable.